mlrun/db/httpd.py

```diff
@@ -472,8 +473,11 @@ def get_files():
         return json_error(HTTPStatus.NOT_FOUND, path=path,
                           err='illegal path prefix or schema')
 
+    secrets = {
+        'V3IO_ACCESS_KEY': access_key,
+    }
     try:
-        body = get_object(path, size, offset)
+        body = get_object(path, secrets, size, offset)
```

Comment:
Where is the change to `get_object`?

Reply:
This is the `get_object` function signature:
`def get_object(url, secrets=None, size=None, offset=0, db=None):`
This call to it was broken (size went into secrets etc...), I've fixed it.